pkg/api/latest/latest.go

```diff
@@ -160,6 +160,9 @@ func init() {
 	// enumerate all supported versions, get the kinds, and register with the mapper how to address our resources
 	for _, version := range versions {
 		for kind := range api.Scheme.KnownTypes(version) {
+			if _, err := kubeMapper.RESTMapping(kind, version); err == nil {
```

Comment:
Yes, Nodes were being assigned to namespace scope.  Because we wrap, we can't register all types, just ours (otherwise we'd have to encode upstream rules).

> On May 7, 2015, at 9:09 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/api/latest/latest.go:
> 
> > @@ -160,6 +160,9 @@ func init() {
> >  	// enumerate all supported versions, get the kinds, and register with the mapper how to address our resources
> >  	for _, version := range versions {
> >  		for kind := range api.Scheme.KnownTypes(version) {
> > +			if _, err := kubeMapper.RESTMapping(kind, version); err == nil {
> Significant change. Were we bugged before?
> 
> —
> Reply to this email directly or view it on GitHub.
>